### PR TITLE
Add pull request template and update README.md

### PR DESCRIPTION
**Pull request by KwikKill**
Update Readme
Add pull request template

### CODE REVIEW

File: docs/manuel/src/03-existant/modules/tickets.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# tickets
```
Comment (Lugrim) [inline docs/manuel/src/03-existant/modules/tickets.md:1]: Majuscule au début ?

File: docs/manuel/src/03-existant/modules/pizza.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
+# pizza
```
Comment (Lugrim) [inline docs/manuel/src/03-existant/modules/pizza.md:1]: Majuscule au début ?

File: docs/manuel/src/03-existant/modules/payment.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
+# payment
```
Comment (Lugrim) [inline docs/manuel/src/03-existant/modules/payment.md:1]: Majuscule au début ?

**Review by Lugrim — APPROVED** [review]
This is some god damn great work, it was missing so much with the previous website <3

File: docs/manuel/src/03-existant/modules/pizza.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+
+Vous sentez cette bonne odeur de fromage et de tomate ? C'est le module `pizza`
+qui arrive ! Ce module permet de gérer les commandes de pizza pour les
+événements de l'association. Il permet de créer des types de pizzas, de les
+commander, de les annuler, etc.
+
+## Pizzas
+
+Une pizza, c'est quoi ? Bah pour nous c'est juste un nom, une image, des
+ingrédients et des alergènes. C'est pas plus compliqué que ça. Et si vous êtes
```
Comment (ShiroUsagi-san) [inline docs/manuel/src/03-existant/modules/pizza.md:11]: allergènes

**Review by ShiroUsagi-san — COMMENTED** [review]
This PR is one of the most important addition of this website's version. Very good introduction, we need to keep this documentation up to date!

**Pull request merged**